src/db.c

```diff
@@ -1437,6 +1438,22 @@ long long getExpire(redisDb *db, robj *key) {
     return dictGetSignedIntegerVal(de);
 }
 
+/* Delete the specified expired keyobj and propagate expires. */
+void expireKeyAndPropagate(redisDb *db, robj *keyobj) {
```

Comment:
```suggestion
/* Delete the specified expired key and propagate expire. */
void deleteExpiredKeyAndPropagate(redisDb *db, robj *keyobj) {
```
maybe `deleteExpiredKeyAndPropagate` is more accurate

Reply:
nice change!